### PR TITLE
Add more doc

### PR DESCRIPTION
**Pull request by antoine-de**

### CODE REVIEW

File: source/ed/readme.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Every executable needs a ```--connection-string``` option to be able to be linked to the database
+
+This connection string must contains:
+ * host: database host
+ * user: database user
+ * password: database password
+ * dbname: name of the database
+
+example: ```--connection-sting="host=localhost user=my_user password=my_pwd dbname=my_db"```
```
Comment (TeXitoi) [inline source/ed/readme.md:24]: Add the fact that importing data will remove the data of the specified type (i.e. osm2ed and geopal2ed will remove street network data).

File: source/ed/readme.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+## nav2rt
+Component that take a kraken input file in input and load real time data into it.
+
+## fare2ed
+Component that load fare data into `ed`.
+
+Note: fares can also be loaded with fusio2ed if the fare files are in the fusio data directory
+
+## poi2ed
+Component that load poi into `ed`.
```
Comment (TeXitoi) [inline source/ed/readme.md]: Add (Point Of Interest)

File: source/ed/readme.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## geopal2ed
+Component that load geopal street network data into `ed`.
+
+## synomyme2ed
+Component that load synomyme data into `ed`.
```
Comment (TeXitoi) [inline source/ed/readme.md]: A small line explaining what synonyme is for.

File: source/ed/readme.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Note: fares can also be loaded with fusio2ed if the fare files are in the fusio data directory
+
+## poi2ed
+Component that load poi into `ed`.
+
+## geopal2ed
+Component that load geopal street network data into `ed`.
+
+## synomyme2ed
```
Comment (l-vincent-l) [inline source/ed/readme.md]: nym*

File: source/ed/readme.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+## fare2ed
+Component that load fare data into `ed`.
+
+Note: fares can also be loaded with fusio2ed if the fare files are in the fusio data directory
+
+## poi2ed
+Component that load poi (Poi Of Interest) into `ed`.
+
+## geopal2ed
+Component that load geopal street network data into `ed`.
```
Comment (l-vincent-l) [inline source/ed/readme.md]: loads

File: source/ed/readme.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+## nav2rt
+Component that take a kraken input file in input and load real time data into it.
+
+## fare2ed
+Component that load fare data into `ed`.
+
+Note: fares can also be loaded with fusio2ed if the fare files are in the fusio data directory
+
+## poi2ed
+Component that load poi (Poi Of Interest) into `ed`.
```
Comment (l-vincent-l) [inline source/ed/readme.md]: loads

File: source/ed/readme.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+Fusio is a CanalTP custom file format derived from gtfs but with additional files.
+
+The directory containing all fusio text files is given with the `-i` option
+
+## nav2rt
+Component that take a kraken input file in input and load real time data into it.
+
+## fare2ed
+Component that load fare data into `ed`.
```
Comment (l-vincent-l) [inline source/ed/readme.md]: loads

File: source/ed/readme.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+## fusio2ed
+Component that load a fusio data set into `ed`.
+
+Fusio is a CanalTP custom file format derived from gtfs but with additional files.
+
+The directory containing all fusio text files is given with the `-i` option
+
+## nav2rt
+Component that take a kraken input file in input and load real time data into it.
```
Comment (l-vincent-l) [inline source/ed/readme.md]: takes, loads

File: source/ed/readme.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+## osm2ed
+Component that load a osm .pbf file into `ed`
+
+## gtfs2ed
+Component that load a gtfs data set into `ed`.
+
+The directory containing all gtfs text files is given with the `-i` option
+
+## fusio2ed
+Component that load a fusio data set into `ed`.
```
Comment (l-vincent-l) [inline source/ed/readme.md]: loads

File: source/readme.md [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+### third_party
+Various third party libraries
+
+### linenoise
+submodule with a lib made to create prompt.
+
+Used to a a debug cli for kraken
+
+### lz4_filter
+Module to handle lz4 compresion format in boost::serialize
```
Comment (TeXitoi) [inline source/readme.md]: CompreSSion

File: source/ed/readme.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+Note: importing data will remove the data of the specified type (i.e. osm2ed and geopal2ed will remove street network data).
+
+## ed2nav
+Component that aggregate all data from `ed` and build the kraken input file.
+
+To run this component, some public transport data *must* be loaded in the database (but other data are not mandatory)
+
+## osm2ed
+Component that load a osm .pbf file into `ed`
```
Comment (l-vincent-l) [inline source/ed/readme.md]: loads

File: source/ed/readme.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+ * user: database user
+ * password: database password
+ * dbname: name of the database
+
+example: ```--connection-sting="host=localhost user=my_user password=my_pwd dbname=my_db"```
+
+Note: importing data will remove the data of the specified type (i.e. osm2ed and geopal2ed will remove street network data).
+
+## ed2nav
+Component that aggregate all data from `ed` and build the kraken input file.
```
Comment (l-vincent-l) [inline source/ed/readme.md]: aggregates

File: source/readme.md [hunk 13]
```diff
@@ -0,0 +1,195 @@
...
+
+Kraken use data serialized with boost::serialize (and compressed with lz4)
```
Comment (TeXitoi) [inline source/readme.md]: useS

File: source/readme.md [hunk 14]
```diff
@@ -0,0 +1,195 @@
...
+This module contains tool to read GTFS, Fusio (format used by CanalTP, derived from GTFS), Osm, ...
+
+It also contains ed2nav, the tool to build all those data together for Kraken
+
+:zap: **cpp module**
+
+### georef
+Handle the street network in kraken.
+
+Contains the shortest path algorithms and some tools explore the street network data (address, house numbers, ...)
```
Comment (l-vincent-l) [inline source/readme.md]: and some tools _to_

Comment (l-vincent-l) [inline source/readme.md]: addresses

File: source/readme.md [hunk 15]
```diff
@@ -0,0 +1,195 @@
...
+
+:zap: **cpp module**
+
+### jormungandr
+Web service around the c++ core(s).
+
+This is the navitia api front end, based over flask
```
Comment (l-vincent-l) [inline source/readme.md]: I think based on is better

Comment (TeXitoi) [inline source/readme.md]: Or built over?

File: source/ed/readme.md (hunk 1 above)
Comment (antoine-de) [inline source/ed/readme.md:24]: done

**Comment by antoine-de** [comment]
close #6 

better late than never :smile:

**Pull request merged**